### ISSUE TITLE
[Bug] Videos always end without continuing to the next one

### ISSUE THREAD

**Issue by Sambammers** [DeathWrench/BestestTVMod#11]
Haven't been able to get the mod to play videos sequentially and automatically once a clip ends regardless of my config settings.

It's very frustrating having to manually switch it to the next clip.

**Comment by DeathWrench** [comment]
You have TVLoader installed as well?

**Comment by Cuzziwuzzi** [comment]
Yeah there seems to be quite a few issues with the mod right now. Random skipping of videos, I had a video completely lock up the TV and break it for the entire session, and some people can't even load any videos

**Comment by GeneralShrooms** [comment]
Same Issue, videos stop and freeze at the last frame without moving on to the next channel

**Comment by DeathWrench** [comment]
If someone makes a PR that fixes it, I'll merge it and upload it to Thunderstore. I haven't played the game in a while though.
